core/src/main/java/bisq/core/trade/HavenoUtils.java

```diff
@@ -110,14 +110,20 @@ public static double atomicUnitsToXmr(BigInteger atomicUnits) {
         return new BigDecimal(atomicUnits).divide(new BigDecimal(XMR_AU_MULTIPLIER)).doubleValue();
     }
 
+    public static String formatToXmr(Coin xmrAmount) {
+        return XMR_FORMATTER.format(coinToXmr(xmrAmount));
+    }
+
     public static BigInteger xmrToAtomicUnits(double xmr) {
         return BigDecimal.valueOf(xmr).multiply(new BigDecimal(XMR_AU_MULTIPLIER)).toBigInteger();
     }
 
     public static long xmrToCentineros(double xmr) {
         return atomicUnitsToCentineros(xmrToAtomicUnits(xmr));
     }
-    
+    public static double coinToXmr(Coin coin) {
```

Comment:
Needs newline above to preserve formatting.

Reply:
This still needs a newline above...

Reply:
New line added.